try{
    var b = 1;
}catch(e);
println(b); // should get error here
println("Should have gotten an error by now :(");